# --- T2-COPYRIGHT-BEGIN ---
# t2/package/*/sox/hotfix-dolbyb.patch
# Copyright (C) 2025 The T2 SDE Project
# SPDX-License-Identifier: GPL-2.0 or patched project license
# --- T2-COPYRIGHT-END ---

libdolbyb.c:28:10: fatal error: configure.h: No such file or directory
   28 | #include "configure.h"  /* for WORDS_BIGENDIAN */

--- sox_ng-14.6.0/libdolbyb/libdolbyb.c.vanilla	2025-06-22 20:43:10.756997485 +0200
+++ sox_ng-14.6.0/libdolbyb/libdolbyb.c	2025-06-22 20:43:19.584010131 +0200
@@ -25,7 +25,7 @@
 #include "SidePath.h"
 #include "FindOutSmp.h"
 
-#include "configure.h"  /* for WORDS_BIGENDIAN */
+#include "soxconfig.h"  /* for WORDS_BIGENDIAN */
 
 #include <stdlib.h>  /* for free() */
 #include <string.h>  /* for memset() */
